platform/linux-generic/odp_classification.c

```diff
@@ -80,102 +90,46 @@ pmr_t *get_pmr_entry_internal(odp_pmr_t pmr)
 
 int odp_classification_init_global(void)
 {
-	odp_shm_t cos_shm;
-	odp_shm_t pmr_shm;
-	odp_shm_t queue_grp_shm;
+	odp_shm_t shm;
 	int i;
 
-	cos_shm = odp_shm_reserve("_odp_shm_odp_cos_tbl",
-				  sizeof(cos_tbl_t),
-				  sizeof(cos_t),
-				  _ODP_SHM_NO_HP);
+	shm = odp_shm_reserve("_odp_cls_global", sizeof(cls_global_t),
+			      ODP_CACHE_LINE_SIZE, 0);
```

Comment:
that is small mapping, why did you remove _ODP_SHM_NO_HP here?

Reply:
Matias will send shortly PR which removes/replaces the flag and changes HP mapping. We did run into latency problems with the current NO_HP mappings. The PR will suggest to use normal pages for <64kB mappings, and HP for larger ones.

Also another aspect to this is number of page table entries used. The smaller the pages size the more page table entries are used, and it's more likely to run into TLB misses (which hurt performance and latency). 

This struct is about 150kB (38x4kB pages vs one 2MB page).